ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -802,3 +817,29 @@ func (r *RayServiceReconciler) reconcileServe(ctx context.Context, rayServiceIns
 
 	return ctrl.Result{}, true, nil
 }
+
+func (r *RayServiceReconciler) labelServingPods(ctx context.Context, rayClusterInstance *rayv1alpha1.RayCluster) error {
```

Comment:
Maybe labelHealthyServingPods or something? Something to indicate we're doing a health-check in this function>
A comment documenting this function would be good.